### ISSUE TITLE
'Pinch to zoom' causes the canvas to disappear

### ISSUE THREAD

**Issue by JustThemo** [drawpile/Drawpile#975]
I have no clue what's going on but I hope I can at least provide the steps.

I've connected my iPad to my PC so I could draw with the stylus but when I use my fingers to zoom in and accidentaly swipe both of them at the same time the canvas disappears and the window in which it should be goes white. I can't draw on it with the stylus or the mouse. It's like there's no canvas at all. I need to zoom out using the size slider to make the canvas reappear but I still need to scroll around to find where it is exactly. When I disable pinch to zoom I don't get this problem but it's clunky to zoom with the mouse and rotate with my fingers.

TL;DR
1. Connect tablet to a PC and run Drawpile
2. Have pinch to zoom enabled
3. Swipe with both fingers in one direction
4. Canvas goes blank
